### PR TITLE
Update visual elements on home page

### PR DESCRIPTION
**Pull request by oomIRL**
@jenreiher-bcgov Updating visual elements for DEVEX-625

### CODE REVIEW

**Comment by MonicaG** [comment]
@jenreiher-bcgov

File: packages/app/src/components/home/HomePage.tsx [hunk 1]
```diff
@@ -47,10 +49,20 @@ const HomePage = () => {
 		}
 
 		body {
-			padding: 1rem 3rem;
+			padding: 2.1rem 9%;
```
Comment (jenreiher-bcgov) [inline packages/app/src/components/home/HomePage.tsx:52]: usually we'd do sensible increments on rems - 2 or 1.5. but not a huge deal

File: packages/app/src/components/home/HomePage.tsx [hunk 2]
```diff
@@ -98,88 +110,77 @@ const HomePage = () => {
...
-						<ContainerImg
-							src="https://digital.gov.bc.ca/wp-content/uploads/2023/03/digitalFrameworkGrey.png"/>
-						<ContainerH2>Get started with mobile development.</ContainerH2>
-						<ContainerP>The BC Mobile Developer Guide provides guidance to developers on steps and practices
-							to
-							follow for developing mobile application for the BC government as well as the supports
-							available to
-							them.</ContainerP>
+				<Grid item sm={12} md={4} style={{display: 'flex'}}>
+					<Container style={{display: 'flex', flexDirection: 'column', justifyContent: 'flex-start'}}>
```
Comment (jenreiher-bcgov) [inline packages/app/src/components/home/HomePage.tsx:144]: please use the MUI sx API for this instead next time
https://mui.com/system/flexbox/

File: packages/app/src/components/home/HomePage.tsx [hunk 3]
```diff
@@ -98,88 +110,77 @@ const HomePage = () => {
...
-					<Container>
-						<ContainerImg src="https://digital.gov.bc.ca/wp-content/uploads/2023/04/tools-2.png"/>
-						<ContainerH2>Get up to speed on application development.</ContainerH2>
-						<ContainerP>The BC Developer Guide covers everything developers need to know in order to build
-							quality,
-							consistent, and compliant applications for the BC government as well as the supports
-							available to
-							them.</ContainerP>
+				<Grid item sm={12} md={4} style={{display: 'flex'}}>
+					<Container style={{display: 'flex', flexDirection: 'column', justifyContent: 'flex-start'}}>
```
Comment (jenreiher-bcgov) [inline packages/app/src/components/home/HomePage.tsx:160]: same here

**Pull request merged**